src/rest-server/src/models/v2/user.js

```diff
@@ -22,6 +22,24 @@ const groupModel = require('@pai/models/v2/group');
 
 const crudType = 'k8sSecret';
 const crudUser = crudUtil.getStorageObject(crudType);
+const keygen = require('ssh-keygen');
+const deasync = require('deasync');
+
+function getSSHKey() {
+  let ret;
+  const location = 'id_rsa';
+  keygen({ location: location }, function (err, result) {
```

Comment:
How about using native nodeJS library https://stackoverflow.com/questions/8520973/how-to-create-a-pair-private-public-keys-using-node-js-crypto

Reply:
https://stackoverflow.com/a/52775583

As you can see from the comment area of this answer, even if you use crypto, you still have to rely on the third-party package node-forge. I have also done relevant tests, and this solution is not feasible.

Reply:
We can rely on node-forge since it's a popular library. But ssh-keygen only has few stars, we don't know if the library still be maintained.

Reply:
1. I have done relevant tests, and the solution provided by [https://stackoverflow.com/a/52775583](https://stackoverflow.com/a/52775583) using `crypto` and  `node-forge` is not feasible.
2. node v10 doesn't seem to support crypto.generateKeyPairSync.